fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -138,9 +139,10 @@ public void getJobs(
 
             JobsApiRequest apiRequest = new JobsApiRequest (
                     format,
-                    null, //asynAfter is null so it behaves like a synchronous request
+                    null, //asyncAfter is null so it behaves like a synchronous request
```

Comment:
To be clear, this isn't a ding against this PR, but rather prior ApiRequest changes around asyncAfter that might need to be revisited.